ansible_collections/arista/avd/extensions/molecule/eos_cli_config_gen/documentation/devices/host1.md

```
@@ -9689,6 +9689,8 @@ ASN Notation: asdot
 | bgp additional-paths send ecmp limit 30 |
 | update wait-for-convergence |
 | update wait-install |
+| bgp convergence time | 400
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)